rmf_site_editor/src/site/model.rs

```diff
@@ -659,3 +778,60 @@ pub fn propagate_model_property<Property: Component + Clone + std::fmt::Debug>(
         }
     }
 }
+
+/// This system keeps model instances up to date with the properties of their affiliated descriptions
+pub fn update_model_instances<T: Component + Default + Clone>(
+    mut commands: Commands,
+    model_properties: Query<
+        (Entity, &NameInSite, Ref<ModelProperty<T>>),
```

Comment:
`Entity, &NameInsite` are unusued and can be removed from the query

Reply:
[661ecb9](https://github.com/open-rmf/rmf_site/pull/242/commits/661ecb96697def25d13fe57e9c252a4ecd388c91)